.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Require keyword arguments for register_custom_action
+19251506f734117a564c59e157221f37e5cf97ef
```

Comment:
Moved to d74545a309ed02fdc8d32157f8ccb9f7559cd185 after rebase, this should hopefully stay after merge:

```suggestion
d74545a309ed02fdc8d32157f8ccb9f7559cd185
```